index.js

```diff
@@ -12,16 +14,20 @@ module.exports = {
   },
   schema: {
     group: {
-      init: groupInitSchema,
+      initRoot: groupInitRootSchema,
+      initEpoch: groupInitEpochSchema,
       addMember: groupAddMemberSchema,
-      content: groupContentSchema
+      content: groupContentSchema,
+      exclude: groupExcludeSchema
     }
   },
   validator: {
     group: {
-      init: Validator(groupInitSchema),
+      initRoot: Validator(groupInitRootSchema),
```

Comment:
:fire: this is a breaking change (the API) is different. I think if we've not published this v2 init then it's fine, but otherwise we need to make this a major version change... or provide an `init` which is the same as `initRoot` ?

Reply:
yeah none of the new group/tribes2 stuff has been in production yet so this repo is still essentially "groups 2.0-release candidate. when it is in production we'll have to move it to the sip repo to freeze it